express-notes-api/getAll.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+const file = require('./data.json');
+const message = require('./messages.json');
+
+function getAllNotes(req, res) {
+  fs.readFile('./data.json', 'utf8', err => {
+    if (err) {
+      res.status(500).json(message.generic);
+    } else {
+      const notesArray = [];
+      for (const id in file.notes) {
+        notesArray.push(file.notes[id]);
+      }
+      res.status(200).json(notesArray);
+    }
+  });
```

Comment:
You can skip reading the file since you already `require()`d its content on line 2.